orchagent/intfsorch.cpp

```diff
@@ -100,6 +100,74 @@ set<IpPrefix> IntfsOrch:: getSubnetRoutes()
     return subnet_routes;
 }
 
+bool IntfsOrch::setIntf(Port& port, sai_object_id_t vrf_id, IpPrefix *ip_prefix)
```

Comment:
Suggest to keep this in the original code and not new function. Its hard to find what changed in new code.